public/xmlns/content.xsd

```diff
@@ -435,7 +435,9 @@
                         </xs:sequence>
                         <xs:attribute name="listeners" type="content:listeners">
                             <xs:annotation>
-                                <xs:documentation>event_ids that trigger this tab</xs:documentation>
+                                <xs:documentation>event_ids that trigger this tab. These listeners are only "active"
```

Comment:
When you describe "active" do you mean, they are callable?

Reply:
yes, that means that the listener is actively listening for events, if the tabs UI is on a different page it is not actively listening for events until that page becomes active. Another word I could have used was enabled